draft-ietf-quic-transport.md

```diff
@@ -4586,15 +4586,15 @@ The three ECN Counts are:
 
 ECT(0) Count:
 : A variable-length integer representing the total number of packets received
-  with the ECT(0) codepoint.
+  with the ECT(0) codepoint in the same packet number space of the ACK.
 
 ECT(1) Count:
 : A variable-length integer representing the total number of packets received
-  with the ECT(1) codepoint.
+  with the ECT(1) codepoint in in the same packet number space of the ACK.
 
 CE Count:
 : A variable-length integer representing the total number of packets received
-  with the CE codepoint.
+  with the CE codepoint in the same packet number space of the ACK.
```

Comment:
```suggestion
  with the CE codepoint in the packet number space of the ACK frame.
```

Reply:
Thanks for the suggestion! I've fixed this one along with the other two in the latest commit.